### PR TITLE
Add comment to generate go doc

### PR DESCRIPTION
**Pull request by Kafakk**

### CODE REVIEW

File: libs/client/client.go [hunk 1]
```diff
@@ -43,6 +46,7 @@ func NewDefaultTestNetClient(veloNodeUrl string, stellarAccountSecretKey string)
 	return NewTestNetClient(grpcConn, stellarAccountSecretKey)
 }
 
+// New public client is a custom client to connect to velo public network.
```
Comment (jura-b) [inline libs/client/client.go]: NewPublicClient creates a new public network client instance to with custom GRPC connection

File: libs/client/client.go [hunk 2]
```diff
@@ -25,6 +26,7 @@ type Client struct {
 	grpcConnection *grpc.ClientConn
 }
 
+// New default public client is a default client to connect to velo public network.
```
Comment (jura-b) [inline libs/client/client.go]: NewDefaultPublicClient creates a default public network client instance.

File: libs/client/client.go [hunk 3]
```diff
@@ -61,6 +66,7 @@ func NewTestNetClient(grpcConn *grpc.ClientConn, stellarAccountSecretKey string)
 	return newClient(horizonclient.DefaultTestNetClient, network.TestNetworkPassphrase, cenGrpc.NewVeloNodeClient(grpcConn), grpcConn, keyPair), nil
 }
 
+// New client is a custom client to connect to velo custom network.
```
Comment (jura-b) [inline libs/client/client.go]: NewClient creates a custom Velo Node client

File: libs/client/client.go [hunk 4]
```diff
@@ -75,19 +81,23 @@ func newClient(horizonClient horizonclient.ClientInterface, networkPassphrase st
 	}
 }
 
+// Close grpc connection
 func (client *Client) Close() (err error) {
 	return client.grpcConnection.Close()
 }
 
+// Set a key pair, which will be used to sign Stellar transaction
```
Comment (jura-b) [inline libs/client/client.go]: SetKeyPair is a helper function that sets the `keyPair` of `Client`

File: libs/client/client.go [hunk 5]
```diff
@@ -75,19 +81,23 @@ func newClient(horizonClient horizonclient.ClientInterface, networkPassphrase st
...
 	client.keyPair = keyPair
 }
 
+// Whitelist result struct contains success result from client.
 type WhitelistResult struct {
 	HorizonResult  *horizon.TransactionSuccess
 	VeloNodeResult *cenGrpc.WhitelistOpResponse
 }
 
+// Whitelist calling velo node to perform whitelist operation
```
Comment (jura-b) [inline libs/client/client.go]: calls

File: libs/client/client.go [hunk 6]
```diff
@@ -101,11 +111,13 @@ func (client *Client) Whitelist(ctx context.Context, veloOp vtxnbuild.Whitelist)
 	}, err
 }
 
+// Setup credit result struct contains success result from client.
```
Comment (jura-b) [inline libs/client/client.go]: SetupCreditResult struct contains transaction result of SetupCredit operation

File: libs/client/client.go [hunk 7]
```diff
@@ -101,11 +111,13 @@ func (client *Client) Whitelist(ctx context.Context, veloOp vtxnbuild.Whitelist)
...
 type SetupCreditResult struct {
 	HorizonResult  *horizon.TransactionSuccess
 	VeloNodeResult *cenGrpc.SetupCreditOpResponse
 }
 
+// Setup credit calling velo node to perform setup credit operation
```
Comment (jura-b) [inline libs/client/client.go]: SetupCredit submits a VeloTx that contains SetupCreditOp to Velo Node and submits the result XDR to Horizon server.

File: libs/client/client.go [hunk 8]
```diff
@@ -245,10 +265,12 @@ func (client *Client) executeVeloTx(ctx context.Context, veloOp vtxnbuild.VeloOp
 	return &result, reply, nil
 }
 
+// Get exchange rate calling velo node to perform get exchange rate returns exchange rate information.
```
Comment (jura-b) [inline libs/client/client.go]: GetExchangeRate returns exchange rate data from Velo

File: libs/client/client.go [hunk 9]
```diff
@@ -245,10 +265,12 @@ func (client *Client) executeVeloTx(ctx context.Context, veloOp vtxnbuild.VeloOp
...
 func (client *Client) GetExchangeRate(ctx context.Context, request *cenGrpc.GetExchangeRateRequest) (*cenGrpc.GetExchangeRateReply, error) {
 	return client.veloNodeClient.GetExchangeRate(ctx, request)
 }
```
Comment (jura-b) [inline libs/client/client.go:278]: GetCollateralHealthCheck returns collateral reserve data from Velo

File: libs/client/client.go [hunk 10]
```diff
@@ -16,15 +18,22 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+// Client struct contains data that use to connect to Velo Node and Stellar.
```
Comment (velo-tech) [inline libs/client/client.go]: Client struct contains data for connecting to the Velo Node and Stellar.

File: libs/client/client.go [hunk 11]
```diff
@@ -16,15 +18,22 @@ import (
...
 type Client struct {
-	horizonClient     horizonclient.ClientInterface
+	// a Horizon client instance which is used to connect to Horizon server
+	horizonClient horizonclient.ClientInterface
+	// the pass phrase used for every Stellar transaction
```
Comment (velo-tech) [inline libs/client/client.go]: The passphrase used for every Stellar transaction

**Review by toppy21 — APPROVED** [review]

**Review by athiwatp — APPROVED** [review]

**Pull request merged**